.github/workflows/windows-qt.yml

```diff
@@ -4,6 +4,7 @@
 name: Windows-Qt
 
 on:
+  workflow_dispatch:
```

Comment:
Why did you add that?

Reply:
I added this line of code to be able to run workflow manually, which is not necessary in this pr. I chose the wrong branch when I submitted this pr, I will close the current pr and submit a new one. #699 

Thank you.